### ISSUE TITLE
[Suggestion] Une vue globale encore plus synthétique (comme le regretté mulitboards.net)

### ISSUE THREAD

**Issue by carotx** [FreshRSS/FreshRSS#3230]
Bonjour, 

Je ne sais pas si vous connaissiez le site multiboards.net (plus en ligne, hélas) ? Il agrégeait une dizaine de flux rss et les affichait de la manière la plus synthétique que j'ai pu trouver jusqu'à présent. 

Appliqué à freshrss, ce serait la vue globale, qui au lieu d'afficher les noms des flux dans chaque catégorie, afficherait directement les titres des articles. 

Il me semble qu'une image vaut mieux qu'un long discours (la seule capture d'écran que j'ai pu trouver encore) :
![image](https://user-images.githubusercontent.com/58709989/96297401-1774d600-0ff1-11eb-976e-2d416ce8dfde.png)

Cela serait-il faisable ? Hélas, je ne sais pas coder pour un sous ... 

En tout cas bravo pour votre travail remarquable.

**Comment by Alkarex** [comment]
Oui, ça serait possible, mais requiert de coder une nouvelle vue et surtout son style

**Comment by math-GH** [comment]
Could you please support us with a short English translation and a higher resolution screenshot?

**Comment by math-GH** [comment]
I guess it is similar to #3885
